include/common.h

```diff
@@ -22,7 +22,7 @@
 #define DONT_INLINE __attribute__((never_inline))
 
 // TODO - Fix MSL (stddef.h)
-#define offsetof(ST, M) ((size_t) & (((ST*)0)->M))
+#define offsetof(ST, M) ((size_t) & (((ST *)0)->M))
```

Comment:
After this is merged we can try to use this macro in the various List users.

Reply:
Gladly xD